packages/replayio/src/utils/installation/installLatestRelease.ts

```diff
@@ -11,82 +11,91 @@ import { debug } from "./debug";
 import { getLatestRelease } from "./getLatestReleases";
 import { MetadataJSON } from "./types";
 
+function getPendingDownloadMessage(attempt: number) {
+  const prefix = `Downloading from ${link("static.replay.io")}`;
+  let suffix = "";
+  if (attempt > 0) {
+    suffix = `${dim(` (retry ${attempt} of 4)`)}`;
+  }
+
+  return `${prefix}${suffix}...`;
```

Comment:
How do you feel about?
```suggestion
    suffix = dim(`(retry ${attempt} of 4)`);
  }

  return `${prefix}... ${suffix}`;
```

That way the message wouldn't "jump" after retry.

Reply:
good idea 👍